v2.0/delete.md

```diff
@@ -31,10 +31,12 @@ table td:first-child {
 | Parameter | Description |
 |-----------|-------------|
 |`relation_expr` | The name of the table you want to delete rows from.<br><br>Deleting from multiple tables in a single statement is not supported.|
-|`AS name` | Create an alias for the table name, completely hiding its original name. All subsequent references to the table must use its alias.<br><br>Aliases are primarily used with `JOIN`, which is not yet supported but is coming in a [future release](https://github.com/cockroachdb/cockroach/issues/2970).|
+|`AS table_alias_name` | Create an alias for the table name, completely hiding its original name. All subsequent references to the table must use its alias.<br><br>Aliases are primarily used with `JOIN`, which is not yet supported but is coming in a [future release](https://github.com/cockroachdb/cockroach/issues/2970).|
```

Comment:
Please remove that second paragraph about JOIN not yet being supported. That's way out of date. Can you also remove it from wherever you copied this text as well?

Reply:
Updated. This was only written here.